README.md

```diff
@@ -39,7 +43,33 @@ It's 100% Open Source and licensed under the [APACHE2](LICENSE).
 ## Usage
 
```

Comment:
Explain how to set `github_oauth_token`

Reply:
![image](https://user-images.githubusercontent.com/52489/51219924-e7fadc00-18e7-11e9-945a-db98bf6b3eec.png)

Reply:
Also, explain that it should not be committed to git and set as an env.